services/stackexchange/stackexchange-monthlyquestions.service.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const BaseJsonService = require('../base-json')
+const { metric } = require('../../lib/text-formatters')
+const { floorCount: floorCountColor } = require('../../lib/color-formatters')
+const moment = require('moment')
+const Joi = require('joi')
+
+const tagSchema = Joi.object({
+  total: Joi.number()
+    .min(1)
+    .required(),
+}).required()
+
+module.exports = class StackExchangeMonthlyQuestions extends BaseJsonService {
+  static get category() {
+    return 'other'
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'stackoverflow' }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Stack Exchange monthly questions',
+        namedParams: { stackexchangesite: 'stackoverflow', query: 'momentjs' },
+        staticExample: this.render({
+          stackexchangesite: 'stackoverflow',
+          numValue: 2000,
+        }),
+        keywords: ['stackexchange', 'stackoverflow'],
+      },
+    ]
+  }
+
+  static get route() {
+    return {
+      base: 'stackexchange',
+      pattern: ':stackexchangesite/monthlyquestions/:query',
+    }
+  }
+
+  static render({ stackexchangesite, numValue }) {
+    const label = `${stackexchangesite} questions`
+
+    return {
+      label,
+      message: `${metric(numValue)}/month`,
+      color: floorCountColor(numValue, 1000, 10000, 20000),
```

Comment:
For consistency with the "total questions" badge, could you include the query in the label?

It seems like this and StackExchangeQuestions could then share a `renderQuestionsBadge` helper function. If the labels are made the same, the only difference is the suffix, which could be another parameter. Then you could call it from both services like this:

```js
static render(props) {
  return renderDownloadBadge({
    interval: 'month',
    ...props
  })
}
```